dev/plugin-guide/en/categories.md

```diff
@@ -217,6 +217,9 @@ Themes control the design and layout of a journal, press or preprint server. Rea
 
 Generic plugins are loaded with every request. They hook into the application early in the [Request Lifecycle](/dev/documentation/en/architecture-request) and can be used to modify almost everything.
 
+> **New in 3.5:** To extend the editorial backend UI with Vue.js components, see [Extending Backend UI with Vue.js](./examples-backend-ui).
```

Comment:
I'm not sure it makes sense to include this in the information about plugin categories. This could be removed (or maybe fit elsewhere like in the templates page).